### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/cookbook/vuepress/config.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#492] [Announcements]
# https://theme-hope.vuejs.press/zh/cookbook/vuepress/config.html

配置文件 如果没有任何配置，你的 VuePress 站点仅有一些最基础的功能。为了更好地自定义你的网站，让我们首先在你的文档目录下创建一个 .vuepress 目录，所有 VuePress 相关的文件都将会被放在这里。你的项目结构可能是这样: VuePress 站点的基本配置文件是 .vuepress/config.js ，但也同样支持 TypeScr...

https://theme-hope.vuejs.press/zh/cookbook/vuepress/config.html

**Comment by tianyige21** [comment]
体验效果

**Comment by rgkjhshi** [comment]
此页内容，原来是在右侧，现在在最上面了，请问这个是什么配置，可以调整到右侧吗？
